trecs/models/content.py

```diff
@@ -196,14 +174,14 @@ def __init__(  # pylint: disable-all
             num_users,
             num_items,
             num_items_per_iter,
-            probabilistic_recommendations=False,
+            probabilistic_recommendations=probabilistic_recommendations,
```

Comment:
Noticed this was wrong earlier and I fixed it across all models; `probabilistic_recommendations` is an optional argument for the `__init__` function but I was accidentally always setting it to `False`. (Not a huge problem because we never use it anyway currently)